frontend/src/component/releases/ReleaseManagement/EmptyTemplatesListMessage.tsx

```diff
@@ -1,38 +1,69 @@
+} from '@mui/material';
+import { RELEASE_PLAN_TEMPLATE_CREATE } from '@server/types/permissions';
 import { ReactComponent as ReleaseTemplateIcon } from 'assets/img/releaseTemplates.svg';
+import PermissionButton from 'component/common/PermissionButton/PermissionButton';
+import { Link, useNavigate } from 'react-router-dom';
 
-const NoReleaseTemplatesMessage = styled('div')(({ theme }) => ({
-    textAlign: 'center',
-    padding: theme.spacing(1, 0, 0, 0),
+const Container = styled('article')(({ theme }) => ({
+    paddingBlock: theme.spacing(5),
+    display: 'flex',
+    gap: theme.spacing(2),
+    flexFlow: 'column nowrap',
+    alignItems: 'center',
+    maxWidth: theme.spacing(40),
+    margin: 'auto',
 }));
 
-const TemplatesEasierMessage = styled('div')(({ theme }) => ({
+const CenteredText = styled(Typography)<TypographyProps>({
     textAlign: 'center',
-    padding: theme.spacing(1, 0, 9, 0),
-    color: theme.palette.text.secondary,
-}));
+});
 
-const StyledCenter = styled('div')(({ theme }) => ({
-    textAlign: 'center',
+const Buttons = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexFlow: 'row wrap',
+    columnGap: theme.spacing(3),
+    rowGap: theme.spacing(2),
+    justifyContent: 'center',
 }));
 
-const StyledDiv = styled('div')(({ theme }) => ({
-    paddingTop: theme.spacing(5),
-}));
 export const EmptyTemplatesListMessage = () => {
+    const navigate = useNavigate();
     return (
-        <StyledDiv>
-            <StyledCenter>
-                <ReleaseTemplateIcon />
-            </StyledCenter>
-            <NoReleaseTemplatesMessage>
-                You have no release templates set up
-            </NoReleaseTemplatesMessage>
-            <TemplatesEasierMessage>
-                Make the set up of strategies easier for your
-                <br />
-                teams by creating templates
-            </TemplatesEasierMessage>
-        </StyledDiv>
+        <Container>
+            <ReleaseTemplateIcon />
+            <CenteredText component='h2' variant='h3'>
+                Get started with release templates
+            </CenteredText>
+            <CenteredText component='p' sx={{ color: 'text.secondary' }}>
+                Control your releases with milestones that can be reused by the
+                entire team.
+            </CenteredText>
+            <Buttons>
+                <Button
+                    component={Link}
+                    sx={{ whiteSpace: 'nowrap' }}
+                    to='https://docs.getunleash.io/reference/release-templates'
+                    variant='text'
+                    rel='noopener noreferrer'
+                    target='_blank'
+                >
+                    Read docs
+                </Button>
+                <PermissionButton
+                    sx={{ whiteSpace: 'nowrap' }}
+                    onClick={() => {
+                        navigate('/release-templates/create-template');
+                    }}
```

Comment:
Yes, I hate this too. However, the page already uses buttons for navigation and it wasn't trivial to set the permission button component such that it would remove the href if the link was disabled. So I'm doing it like this for now. (We also have 20+ instances of using buttons for navigation throughout the app).

(Of course, whether it makes sense to "disable" a link in the first place is a conversation worth having, but admittedly, being told you can't do anything on a linked page before trying to follow it is a nice affordance 🤷🏼 )